data/en/test_checkbox_mutually_exclusive.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
👍 to **not** calling this mutually_exclusive_checkbox.json

Reply:
I don't like the name but I do like that it's testing a checkbox that happens to be mutually exclusive. We need to bunch test schemas together based on the component/feature it's for.